modules/home/assets/js/components/top-section.js

```diff
@@ -3,22 +3,22 @@ import Typography from '@elementor/ui/Typography';
 import Button from '@elementor/ui/Button';
 import YoutubeIcon from '../icons/youtube-icon';
 
-const TopSection = ( props ) => {
+const TopSection = ( { ...props } ) => {
 	return (
 		<Paper elevation={ 0 } sx={ { display: 'flex', flexDirection: { xs: 'column', md: 'row' }, p: { xs: 2, md: 4 }, gap: 3 } }>
 			<Stack gap={ 3 }>
 				<Box>
-					<Typography variant="body2">{ __( 'Hi!', 'elementor' ) }</Typography>
-					<Typography variant="h6">{ __( 'Welcome to Elementor', 'elementor' ) }</Typography>
-					<Typography variant="body2" >{ __( 'Get introduced to Elementor by watching our &ldquo;Getting Started&rdquo; video series. It will guide you through the steps needed to create your website. Then click to create your first page.', 'elementor' ) }</Typography>
+					<Typography variant="body2">{ props.topData.title_small }</Typography>
+					<Typography variant="h6">{ props.topData.title }</Typography>
+					<Typography variant="body2" >{ props.topData.description }</Typography>
 				</Box>
 				<Box sx={ { display: 'flex', gap: 1 } }>
-					<Button variant="contained" size="small" href={ props.createNewPageUrl } target="_blank">{ __( 'Create a Page', 'elementor' ) }</Button>
-					<Button variant="outlined" color="secondary" size="small" startIcon={ <YoutubeIcon /> } href={ props.videoUrl } target="_blank">{ __( 'Watch a quick starter', 'elementor' ) }</Button>
+					<Button variant="contained" size="small" href={ props.createNewPageUrl } target="_blank">{ props.topData.button_create_page_title }</Button>
+					<Button variant="outlined" color="secondary" size="small" startIcon={ <YoutubeIcon /> } href={ props.topData.button_watch_url } target="_blank">{ props.topData.button_watch_title }</Button>
 				</Box>
 			</Stack>
 			<Box component="iframe"
-				src={ props.embedUrl }
+				src={ props.topData.youtube_embed_id }
```

Comment:
@elchugreeva Should this not be prefixed with `https://www.youtube.com/watch?v=`
JSON just contains the Video ID.
CC @hein-obox

Reply:
@elchugreeva @louiswol94 Louis is right

Reply:
@louiswol94 good catch, fixed, it is https://www.youtube.com/embed/